### PR TITLE
[STORM-1390] If you find a suitable constructor do NOT need to look for a d…

### PR DESCRIPTION
**Pull request by lispking**
If you find the right constructor, you do not have to look any further.

### CODE REVIEW

**Comment by unsleepy22** [comment]
@lispking regarding the original code, I think it means to give a warning when it meets multiple eligible constructors (sort of like multiple bindings for slf4j), so I don't think we should break on first match.

**Comment by lispking** [comment]
@unsleepy22 i see, but this may make people confused.

File: external/flux/flux-core/src/main/java/org/apache/storm/flux/FluxBuilder.java [hunk 1]
```diff
@@ -395,6 +395,7 @@ private static Constructor findCompatibleConstructor(List<Object> args, Class ta
                     eligibleCount++;
                 }
```
Comment (HeartSaVioR) [inline external/flux/flux-core/src/main/java/org/apache/storm/flux/FluxBuilder.java]: @lispking ```break``` should be here, not there.

**Comment by HeartSaVioR** [comment]
@lispking 
Though I've addressed potential bug point, I'm with @unsleepy22. 
Its role is not that we strongly need to optimize. It doesn't affect performance.
Knowing user's intention correctly is the most important point to Flux so finding out missing point and showing warning is vaild operation, I think.

**Comment by lispking** [comment]
@HeartSaVioR Thank reminder, your proposal is appropriate。

**Pull request closed without merging**